src/coordsim/traffic_predictor/traffic_predictor.py

```diff
@@ -0,0 +1,46 @@
+from coordsim.simulation.simulatorparams import SimulatorParams
+from collections import defaultdict
+from math import ceil
+import numpy as np
+import logging
+log = logging.getLogger(__name__)
+
+
+class TrafficPredictor():
+    """
+    Traffic Predictor class
+    Updates traffic dict in metrics module to show upcoming traffic at ingress node.
+    """
+
+    def __init__(self, params: SimulatorParams):
+        self.params = params
+
+    def predict_traffic(self):
+        """
+        Predicts the traffic at ingress nodes based on the current inter_arrival_mean
```

Comment:
Maybe clarify: "Predict" = read future traffic from file. At least at the moment

Reply:
What I had in mind here is that with few minor tweaks, I can also adapt this to also 'predict' the traffic of the next MMP state. I can just relocate where the state is updated for MMP scenarios and this will be able to calculate the ingress traffic of the next state. Let me know if I should also focus on this in the next PR.